README.md

```diff
@@ -0,0 +1,6 @@
+# anti-tourist-trap
```

Comment:
Your README could use a little love here... Perhaps tell us more about the project, you the devs, or how someone is to use this repo?